File: Frontend/SPFrontend/src/app/components/models/DeleteExamDTO.ts
import {ExamDTO} from "./ExamDTO";
import {User} from "./User";

export class DeleteExamDTO {
  exam: ExamDTO;
  user: User;

  constructor(exam: ExamDTO, user: User) {
    this.exam = exam;
    this.user = user;
  }

}

File: Backend/src/main/java/de/hft_stuttgart/winf/proj2/sp/backend/dto/DeleteExamDTO.java
package de.hft_stuttgart.winf.proj2.sp.backend.dto;

import lombok.*;

@AllArgsConstructor
@Getter
@ToString
@EqualsAndHashCode
public class DeleteExamDTO {

    private ExamDto exam;

    private UserDto user;

    public DeleteExamDTO(){}
}
